extensions/reference/views.md

```diff
@@ -91,13 +91,25 @@ Your view should use this information to render and manage its form. It is likel
 
 #### `validate() => boolean`
 
-The `validate` method will be called after the user hits the "Save" button in Launch. It should return a boolean indicating whether the user's input is valid. It's up to you as the extension developer to determine what constitutes valid input since your library module will be acting upon that input.
+The `validate` method will be called after the user hits the "Save" button in Launch.
+
+It should return one of the following:
+* a boolean indicating whether the user's input is valid;
+* a promise to later be resolved with a boolean indicating whether the user's input is valid.
```

Comment:
You've got a semicolon at the end of one and a period at the end of the other. Maybe be consistent with a period or nothing at all?